examples/layers/preprocessing/segmentation/demo_utils.py

```diff
@@ -28,6 +28,7 @@
 def normalize(input_image, input_mask):
     input_image = tf.image.convert_image_dtype(input_image, tf.float32)
     input_image = (input_image - mean) / tf.maximum(std, backend.epsilon())
+    input_image = input_image / 255
```

Comment:
Does this break other demos?

Reply:
Actually this seems incorrect -- we're already scaling by mean and stddev so this shouldn't be done

Reply:
No, I checked other demos as well.

Reply:
In this, the value range of image is greater than 255. As we are setting Augmix value range to be [0, 255], so the demo does not work without making this change. Any better alternative? 🤔